driver-mongodb/src/main/resources/mongodb.md

```diff
@@ -0,0 +1,3 @@
+# MongoDB Driver
+
+This is a driver for MongoDB.
```

Comment:
We should document all the parameters that are special to this activity type.

Reply:
Updated. :ok_hand: